src/scripts.js

```diff
@@ -58,7 +60,13 @@ window.addEventListener("load", function () {
       createRecipeCards(res["2"].recipes);
       activeRecipes = [...res["2"].recipes];
     })
-    .catch((err) => console.log(err));
+    // sorry the network seems busy, please try again later, be sure to make sure your server is running
+    .catch(() => {
+      let userGetErrorMessage =
+        "Sorry the network seems to not be working - make sure the server is running and reload the page";
+      displayErrorMessage(userGetErrorMessage);
+      tagSection.classList.toggle("hidden", true);
```

Comment:
Ah! Okay this is close to what I was getting to yesterday before I gave up. I just needed to connect the HTML, function and then call it in the API for the catch!